feeds/wlan-ap/interAPcomm/src/src/interAPcomm.c

```diff
@@ -0,0 +1,124 @@
+/* SPDX-License-Identifier BSD-3-Clause */
+#include <stdio.h>
+#include <sys/socket.h>
+#include <errno.h>
+#include <arpa/inet.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <pthread.h>
+#include <interAPcomm.h>
+
+/*Receiver socket*/
+int recv_sock = -1;
+
+recv_arg ra;
+
+void *receive_data(void *recv_cb)
+{
+	void *recv_data;
+	int recv_data_len;
+	recv_arg *rcb = recv_cb;
+
+	recv_data = malloc(rcb->len);
+	memset(recv_data, 0, rcb->len);
+
+	for (;;) {
+		if ((recv_data_len = recvfrom(recv_sock, recv_data, rcb->len,
+					      0, NULL, 0)) < 0)
+			printf("recvfrom() failed");
+
+		rcb->cb(recv_data);
+	}
+
+	return(NULL);
+}
+
+int interap_recv(unsigned short port, int (*recv_cb)(void *), unsigned int len)
+{
+	struct sockaddr_in addr;
+	int bcast_perm;
+	pthread_t recv_thread;
+
+	ra.cb = recv_cb;
+	ra.len = len;
+
+	if ( port == 0 || recv_cb == NULL || len == 0) {
+		printf("Error: port=%d, recv_cb =%p, len = %d", port, recv_cb,
+			len);
+		return -1;
+	}
+
+	if ((recv_sock = socket(PF_INET, SOCK_DGRAM, IPPROTO_UDP)) < 0) {
+		printf("socket failed");
+		return -1;
+	}
+
+	/* Set socket to allow broadcast */
+	bcast_perm = 1;
+	if (setsockopt(recv_sock, SOL_SOCKET, SO_BROADCAST, (void *) &bcast_perm,
+			sizeof(bcast_perm)) < 0)
+		printf("setsockopt failed");
+
+	memset(&addr, 0, sizeof(addr));
+	addr.sin_family = AF_INET;
+	addr.sin_addr.s_addr = htonl(INADDR_ANY);  /* Any incoming interface */
+//	addr.sin_addr.s_addr = htonl(INADDR_BROADCAST);
```

Comment:
Please remove commented code from all the files

Reply:
It appears that the comments were not addressed in the latest version of the patch